server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -256,24 +256,50 @@ function createTableCell(index, sortValue, showValue) {
       '">' + showValue + '</td>';
 }
 
+/**
+ * Builds console logging message for REST GET calls
+ * @param {string} REST url called
+ * @param {object} data returned from REST call
+ * @param {string} session storage/global variable to hold REST data
+ */
+ function createRestGetLog(call, data, sessionDataVar){
+    var jsonDataStr = JSON.stringify(data);
+
+    //Handle data to be stored in global variable instead of session storage
+    if (sessionDataVar==="NAMESPACES") {
+      NAMESPACES = jsonDataStr;
+      console.debug("REST GET call to " + call +
+       " stored in " + sessionDataVar + " = " + NAMESPACES);
+    }
+    else {
+      sessionStorage[sessionDataVar] = jsonDataStr;
+      console.debug("REST GET request to " + call +
+        " stored in sessionStorage." + sessionDataVar + " = " + sessionStorage[sessionDataVar]);
+    }
+ }
+
 ///// REST Calls /////////////
 
 /**
  * REST GET call for the manager information,
  * stores it on a sessionStorage variable
  */
 function getManager() {
-  return $.getJSON('/rest/manager', function(data) {
-    sessionStorage.manager = JSON.stringify(data);
+  var call = '/rest/manager';
+  console.info("Retrieving " + call);
+  return $.getJSON(call, function(data) {
+    createRestGetLog(call, data, 'manager');
   });
 }
```

Comment:
Since the same pattern is used repeatedly, this code could be simplified even further, by replacing it with a new function, something like `getJSONforTable('/rest/manager', 'manager');`